lib/consult/template.rb

```diff
@@ -75,7 +75,7 @@ def contents
 
     def consul_contents(location)
       [@config[location]].compact.flatten.map do |key|
-        Diplomat::Kv.get(key, options: nil, not_found: :return, found: :return)
+        Diplomat::Kv.get(key, options: nil, not_found: :return, found: :return).force_encoding 'utf-8'
```

Comment:
What encoding are you seeing this come back as without this? And did you try just `encode` instead of `force`?

Reply:
I updated the commit message to clarify some more, but it's `ASCII-8BIT`. That's probably because of `Base64.decode64`. The problem is, if you have a unicode string in Consul, then it's escaped when it comes out here, which normally wouldn't be a big deal but Ruby throws an exception when trying to combine these with the UTF-8 strings from files (explicitly set below in `disk_contents`).